### PR TITLE
[storage-ors-preview] Support ORS policy for storage account

### PR DESCRIPTION
**Pull request by Juliehzl**
---

This checklist is used to make sure that common guidelines for a pull request are followed.

### General Guidelines

- [ ] Have you run `azdev style <YOUR_EXT>` locally? (`pip install azdev` required)
- [ ] Have you run `python scripts/ci/test_index.py -q` locally?

For new extensions:

- [ ] My extension description/summary conforms to the [Extension Summary Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/extensions/extension_summary_guidelines.md).

### About Extension Publish

There is a pipeline to automatically build, upload and publish extension wheels.  
Once your PR is merged into master branch, a new PR will be created to update `src/index.json` automatically.  
The precondition is to put your code inside this repo and upgrade the version in the PR but do not modify `src/index.json`.

### CODE REVIEW

**Comment by yonzhan** [comment]
add to S168

File: src/storage-ors-preview/README.md [hunk 1]
```diff
@@ -0,0 +1,170 @@
+# Azure CLI storage-ors-preview Extension #
+This is a extension for storage-ors-preview features.
```
Comment (haroldrandom) [inline src/storage-ors-preview/README.md]:
```suggestion
This is an extension for storage-ors-preview features.
```

File: src/storage-ors-preview/README.md [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+
+### How to use ###
+Install this extension using the below CLI command
+```
+az extension add -s https://azurecliprod.blob.core.windows.net/cli-extensions/storage_ors_preview-0.1.0-py2.py3-none-any.whl
```
Comment (haroldrandom) [inline src/storage-ors-preview/README.md]: Why `az extension add sotrgae-ors-preview`?

File: src/storage-ors-preview/azext_storage_ors_preview/_help.py [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+  - name: Show the properties of specified rule in Object Replication Service Policy.
+    text: az storage account ors-policy rule show -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332"
+"""
+
+helps['storage account ors-policy rule update'] = """
+type: command
+short-summary: Update rule properties to Object Replication Service Policy.
+examples:
+  - name: Update rule properties to Object Replication Service Policy.
+    text: az storage account ors-policy rule remove -g ResourceGroupName -n StorageAccountName --policy-id "04344ea7-aa3c-4846-bfb9-e908e32d3bf8" --rule-id "78746d86-d3b7-4397-a99c-0837e6741332" --prefix-match blobA blobB
```
Comment (haroldrandom) [inline src/storage-ors-preview/azext_storage_ors_preview/_help.py]: Shall we replace this with dummy vaule?

File: src/storage-ors-preview/azext_storage_ors_preview/_params.py [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+    )
+    prefix_math_type = CLIArgumentType(
+        nargs='+', arg_group='Filters',
+        help='Optional. Filter the results to replicate only blobs whose names begin with the specified '
+             'prefix.'
+    )
+    min_creation_time_type = CLIArgumentType(
+        options_list=['--min-creation-time', '-t'], arg_group='Filters', type=get_datetime_type(True),
+        help="Blobs created after the time will be replicated to the destination. It must be in datetime format "
+             "'yyyy'-'MM'-'dd'T'HH':'mm':'ss'Z'. Example: 2020-02-19T16:05:00Z")
```
Comment (haroldrandom) [inline src/storage-ors-preview/azext_storage_ors_preview/_params.py]: yyyy-MM-dd HH:mm:ss? Make more sense to me without `'`

File: src/storage-ors-preview/azext_storage_ors_preview/_help.py (hunk 3 above)
Comment (Juliehzl) [reply src/storage-ors-preview/azext_storage_ors_preview/_help.py]: This id is not a valid one. In this way it is fine now.

File: src/storage-ors-preview/azext_storage_ors_preview/_params.py (hunk 4 above)
Comment (Juliehzl) [reply src/storage-ors-preview/azext_storage_ors_preview/_params.py]: sure.

**Comment by Juliehzl** [comment]
Wait for PM sign off

**Pull request merged**